src/dao/pages/DAOsListPage/DAOsListPage.tsx

```diff
@@ -1,20 +1,61 @@
-import { Text } from "@chakra-ui/react";
-import { CardGridLayout, Page, PageHeader } from "@dex/layouts";
-import { useDAOs } from "@dao/hooks";
-import { useNavigate } from "react-router-dom";
+import { Box, Flex, Input, Select, Tab, TabList, Tabs, Text } from "@chakra-ui/react";
+import { CardGridLayout, NotFound, Page, PageHeader } from "@dex/layouts";
+import { DAOTabs, useDAOs } from "@dao/hooks";
+import { useNavigate, useSearchParams } from "react-router-dom";
 import { PrimaryHeaderButton } from "@dex/components";
 import { DAOCard } from "./DAOCard";
 import { Routes } from "@dao/routes";
-import { DAO } from "@dao/services";
+import { DAO, DAOType } from "@dao/services";
+import { useDexContext } from "@dex/hooks";
+import { Color, Pagination, usePagination } from "shared";
+import { ChangeEvent } from "react";
+import { filterDAOs } from "../utils";
+import { DAOsPerPage } from "@dex/services";
 
 export function DAOsListPage() {
   const daos = useDAOs();
+  const [searchParams, setSearchParams] = useSearchParams();
+  const searchText = searchParams.get("searchText") ?? "";
```

Comment:
URLs need to be user-friendly so they can be easily read when shared in posts, chat, emails, etc.

In order:

1) `myDAOs` => `filter`  - should be "myDAOs" or nothing (no filter param), not a boolean. 
2) `searchText` => `search`
3) `daoType` => `type`

`http://localhost:3000/?filter=myDAOs&search=hedera&type=NFT